src/Features/Core/Portable/EmbeddedLanguages/DateAndTime/LanguageServices/DateAndTimeEmbeddedLanguage.cs

```diff
@@ -18,12 +17,11 @@ internal class DateAndTimeEmbeddedLanguage : IEmbeddedLanguage
     {
         public readonly EmbeddedLanguageInfo Info;
 
-        public ISyntaxClassifier Classifier { get; }
+        public ISyntaxClassifier? Classifier { get; }
```

Comment:
Probably useful to annotate IEmbeddedLanguage to suit, as this annotation presumably wouldn't have been picked up in EmbeddedLanguagesClassifier anyway, right?